app/_src/gateway/plugin-development/wasm.md

```diff
@@ -0,0 +1,165 @@
+---
+title: WebAssembly
+book: plugin_dev
+chapter: 11
+---
+
+# Hello World: Your First Proxy-Wasm Filter
+
+Filters can be written in almost any language that supports WebAssembly as a
+compilation target, but the easiest  and most ergonomic way to get started is by
+using a language which has an existing [Proxy-Wasm](https://github.com/proxy-wasm/spec)
+SDK, such as [Go](https://github.com/tetratelabs/proxy-wasm-go-sdk/),
+[C++](https://github.com/proxy-wasm/proxy-wasm-cpp-sdk/), or [Rust](https://github.com/proxy-wasm/proxy-wasm-rust-sdk/).
```

Comment:
```suggestion
or [Rust](https://github.com/proxy-wasm/proxy-wasm-rust-sdk/).
```

We've only performed significant testing on the Go and Rust SDKs, so let's not point people to the C++ one just yet.

Reply:
Removed